### DISCUSSION TITLE
Frustrated

### DISCUSSION THREAD

**Discussion by KoalityJustin** [thunderclient/thunder-client-support#1296] [General]
Recently starting migrating projects away from Postman to this tool as Postman was so bloated and overpriced it didn't feel worth the money anymore. Was REALLY impressed with Thunder Client and how simple it was to get started and commit collections of API scaffolds/tests to be shared on a project within the codebase rather than needing a separate tool/web tool/interface but it would just be right there inside VSCode.

But now, I'm mid walkthrough with a dev and suddenly find the VSCode extension blocking me from doing anything but suggesting to "Purchase a subscription" or "Login". 

So I try logging in. 
"Can't find team member."
Maybe I have to add myself as a team member?
"You did not purchase subscription yet."
What? I just want to see my existing collections to show a dev how to work with them.

An hour later after digging around I FINALLY find a discussion mentioning that "Git sync" is now behind the subscription. So after finally disabling that option and seeing the VSCode extension go back to normal; my collections are nowhere to be seen in the sidebar anymore. So I try importing the collection:
"Failed to import: 1 files - thunderCollection.json"

I mean c'mon, I understand trying to pay bills and generate revenue. But this was just an unfortunate way to go about it, by locking existing things I was using behind a subscription that just disrupted a whole day of my work. Which I now just also lost money on cause I spent time on that issue and now writing this feedback. So I lost some money today, and now I need to pay to use something I was already using?

New features/upgrades being available behind a subscription would have been a much easier pill to swallow and I most likely would have talked with the devs here and decided on how many people would need seats to pay for.

I don't know that I'll continue using Thunder Client because this change has had a very immediate negative effect on my local dev workflow that has left a sour taste. 

Not sure what the thought process behind this was, but it was a huge miss for me.

**Comment by rangav** [comment]
Hi @KoalityJustin 

We announced subscriptions 90 days ago
https://github.com/rangav/thunder-client-support/releases/tag/v2.7.0

Please follow the guide to migrate to free version - https://github.com/rangav/thunder-client-support/discussions/1286

**Reply by KoalityJustin** [reply]
The needing a subscription isn't my actual issue. That's actually the smallest complaint I have, one that I fully understand and in a lot of cases I pay for subscription services that simplify my dev workflow.

My biggest issue is that existing functionality suddenly becoming unusable without a subscription. That's the change that I fundamentally disagree with that's frustrating me.

**Reply by rangav** [reply]
- If we charge for our 1st version released in 2021 which does not have all the new features we developed in the last 2 years - Users will complain about how can you charge for such a basic software that does not have features.
- After developing all the new features in the last 2 years, if we want to charge now - You are complaining now how can we charge for existing features
- Some users complain about how can you charge for the X or Y feature.
- No matter when we charge users will complain.

It's easy to come up with an argument for not to charge for the software.

We have 2M+ downloads and got only less than 500 dollars in GitHub sponsorship in 2 years. 

Our team is no longer willing to work for free. So we have started charging for our software to pay our salaries and expenses.

**Reply by rangav** [reply]
I am closing this discussion if you have further questions you can contact us using the below link.
- https://www.thunderclient.com/contact
